README.md

```diff
@@ -60,7 +61,9 @@ You will need to run `python manage.py makemigrations` and `python manage.py mig
 
 *It's intended that these fields are temporary for while importing, and can be removed once the content has been imported. [view source](wagtail_wordpress_import/models.py)*
 
-A full example of the suggested page model class
+#### A full example of the suggested page model class
+
+The import default is to import the `post` and `page` content types to an app called `pages` and a model called `PostPage`. Keep that mind when you create your own page model.
```

Comment:
I actually ran into this issue earlier, wasn't too much of a concern as I just had to read info below, but I was thinking we could add `# mysite/pages/model.py` below this [line](https://github.com/torchbox/wagtail-wordpress-import/blob/3c27330258e24a6b52f3d580060f607706bbc9d0/README.md?plain=1#L65) instead?

Reply:
Thanks ben I added in the comment.